tools/Microsoft.WindowsAzure.Build.Tasks/Build.Tasks.Tests/TestPublishProjects/MultiProjectMultiSln/CSProjTestPublish/Class1.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSProjTestPublish
+{
+    public class Class1
+    {
+    }
```

Comment:
Ok I will add it now :)
I am still not sure, the purpose of adding methods to empty classes. These projects are not meant to be consumed by anyone, they are test assets, whose sole purpose is to get packaged and used in testing publishing functionality.